packages/database/test/order_by.test.ts

```diff
@@ -363,9 +363,7 @@ describe('.orderBy tests', function() {
     expect(addedPrevNames).to.deep.equal(expectedPrevNames);
   });
 
-  it('Removing default listener removes non-default listener that loads all data', function(
-    done
-  ) {
+  it('Removing default listener removes non-default listener that loads all data', function(done) {
```

Comment:
Did you make these changes or is this a prettier thing? cc @jshcrowthe

Reply:
prettier.

Reply:
@jshcrowthe Is there any chance you could look into a way of using the git changes to detect which packages have been changed and only run prettier over that directory.

Reply:
Yeah I can do that. The react team has done it so I'll see what I can reuse.

Reply:
I think this may actually be an issue with using the wrong prettier version.  Can you run "yarn" from the root of the repo and then "yarn prepush" and see if it makes a commit that undoes this change?

Reply:
tried,  doesn't seem to help.